src/meenu/exam_2/AllIntSum.java

```diff
@@ -0,0 +1,34 @@
+package meenu.exam_2;
+
+/*WAP to do sum of all the integer from the given string
+
+Input: String str = "12h14i4w8sdc15"
+
+Output: 12+14+4+8+15 = 53
+*/
+
+import java.util.Scanner;
+
+public class AllIntSum {
+	
+	int getSumOfInt(String word) {
+		int sum = 0;
+		for(int index = 0;index<word.length();index++) {
+			if(Character.isDigit(word.charAt(index)))
```

Comment:
You have created very complex logic apart from that you can make it simple dear, 
Try below code:

void sumOfNumberFromGivenString(String usergivenstring) {
		Character ch;
		int sum = 0, num = 0;
		String concat = "";

		for (int index = 0; index < usergivenstring.length(); index++) {
			ch = usergivenstring.charAt(index);
			if (Character.isDigit(ch)) {
				concat = concat + ch;
			} else if (concat != "") {
				num = Integer.parseInt(concat);
				System.out.println("Num is: " + num);
				sum = sum + num;
				concat = "";
			}
		}
		if (concat != "") {
			num = Integer.parseInt(concat);
			System.out.println("Num is: " + num);
			sum = sum + num;
		}
		System.out.println("Sum of numbers are : " + sum);
	}
Also not merging your code because you have added Rahul's file.
So closing your PR.